packages/core/src/contrib/Functions.ts

```diff
@@ -658,6 +658,34 @@ export const compDict = {
     if (pathType === "closed") path.closePath();
     return path.getPath();
   },
+
+  repeatedArcs: (
+    _context: Context,
+    innerStart: ad.Pt2,
+    innerEnd: ad.Pt2,
+    outerStart: ad.Pt2,
+    outerEnd: ad.Pt2,
+    innerRadius: ad.Pt2,
```

Comment:
Why is the radius a vector?

Reply:
Arcs use ellipses, so they are actually `rx, ry`

Reply:
Gotcha; is the intention that you _can_ use this function to create elliptical arcs, or would it be better to change the interface to just take in an `ad.Num` (while otherwise keeping the implementation the same)?

Reply:
I don't really love the fact that we have these functions so the API is inconsequential to me. Once we have some form of collection in Style these should really go away

Reply:
Sounds good!

Reply:
But yes in theory these could be defined on an ellipse with different radii